selfdrive/camerad/cameras/camera_qcom2.c

```diff
@@ -479,15 +479,15 @@ void config_isp(struct CameraState *s, int io_mem_handle, int fence, int request
 
 void enqueue_buffer(struct CameraState *s, int i) {
   int ret;
-  int request_id = s->camera_bufs_metadata[i].frame_id;
+  int request_id = s->request_ids[i];
 
   if (s->buf_handle[i]) {
     release(s->video0_fd, s->buf_handle[i]);
     // wait
-    // struct cam_sync_wait sync_wait = {0};
-    // sync_wait.sync_obj = s->sync_objs[i];
-    // sync_wait.timeout_ms = 100;
-    // ret = cam_control(s->video1_fd, CAM_SYNC_WAIT, &sync_wait, sizeof(sync_wait));
+    struct cam_sync_wait sync_wait = {0};
+    sync_wait.sync_obj = s->sync_objs[i];
+    sync_wait.timeout_ms = 50;
```

Comment:
Isn't 50ms too close? Since we're running at 20Hz?
Or do we want to skip the frame completely if it doesn't come in right in time?

Reply:
it's 200ms really, since we have 4 buffers each. 50 is kind of arbitrary here

Reply:
no wait is fine too, because by the time we call this it should be ready anyways (from the event).